src/install.rs

```diff
@@ -245,7 +245,30 @@ fn write_platform(mountpoint: &Path, platform: &str) -> Result<()> {
     }
 
     eprintln!("Setting platform to {}", platform);
+    edit_bls_entries(mountpoint, |orig_contents: String| {
+        write_platform_in_bls(&orig_contents, platform)
+    })?;
+
+    Ok(())
+}
+
+fn write_platform_in_bls(orig_contents: &str, platform: &str) -> Result<String> {
```

Comment:
`bls_entry_replace_platform()` maybe? Also, I think you can move portions of the inner comment to a docstring here.

Reply:
I went with `bls_entry_write_platform` to match the name of the function `write_platform`. Will update https://github.com/coreos/coreos-installer/pull/268 too to match that naming convention.